### PR TITLE
add ensure logs

### PR DESCRIPTION
**Pull request by mrIncompetent**
**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/resources/ensure.go [hunk 1]
```diff
@@ -95,43 +97,6 @@ func EnsureRoleBinding(data RoleBindingDataProvider, create RoleBindingCreatorDe
 	return nil
 }
 
-// EnsureClusterRoleBinding will create the RoleBinding with the passed create function & create or update it if necessary.
-// To check if it's necessary it will do a lookup of the resource at the lister & compare the existing RoleBinding with the created one
-func EnsureClusterRoleBinding(data ClusterRoleBindingDataProvider, create ClusterRoleBindingCreatorDeprecated, lister rbacv1lister.ClusterRoleBindingLister, client rbacv1client.ClusterRoleBindingInterface) error {
```
Comment (mrIncompetent) [inline api/pkg/resources/ensure.go:100]: Was unused

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

**Pull request merged**